src/helpers.ts

```diff
@@ -61,24 +61,31 @@ function hasNodeExecutable(rootPath: string, executable: string): boolean {
  * @returns {string}
  */
 export function pathToJest(pluginSettings: IPluginSettings) {
+  if (isDefaultSettingForPathToJest(pluginSettings.pathToJest) && isBootstrappedWithCRA(pluginSettings.rootPath)) {
```

Comment:
> If you really would like to introduce a function for essentially checking if the string is empty

It might seem like a trivial thing to make a function for but the function adds value by:
1. clearly communicating what we're doing
2. isolating the default value so it's easier to change
3. providing a seam to test if we're checking the default value properly

A null check often doesn't provide the context that someone needs to understand where the value came from or what the value means. This can be fixed using (something like a Null Object or [Special Case](https://martinfowler.com/eaaCatalog/specialCase.html)), a comment, or a well named variable or function.

I'd like to help build a community that's engaged and empowered to contribute to this extension. It starts with little things like the trivial function or advocating for code that's easier to maintain in the long run. It may be more verbose but it's a small overhead now that should help us all maintain it in the long run.
<br/>

> four lines later you also should use that function

Do you mean to check if the user has changed the value in `if (pluginSettings.pathToJest)`? That's reordered and changed it to `if (isNotDefaultPathToJest(pathToJest))`.
<br/>

> move the if(pluginSettings.pathToJest) up to the top

Great idea 👍.